cromwell-helm/README.md

```diff
@@ -12,7 +12,7 @@ Here is a suggested workflow for testing updates to this chart from local Termin
 3. Now you can edit the helm chart locally in your favorite editor. To update the instance of Cromwell running with your new chart, do the following from the directory in which `cromwell-helm` lives:
    - helm upgrade --namespace=<namepace> <release> cromwell 
    - Note: you can see the namespace in Google Cloud console to the right of the Cromwell service. Release is the same, with “ns” replaced by “rls”.
-     So for example: helm upgrade --namespace=jlfcsv-gxy-ns jlfcsv-gxy-rls cromwell
+     So for example: helm upgrade --namespace=jlfcsv-gxy-ns jlfcsv-gxy-rls cromwell-helm
```

Comment:
Should we also remove the `do the following from the directory in which `cromwell-helm` lives:` comment on line 12.